bundles/preview/preview-form.rst

```diff
@@ -0,0 +1,53 @@
+Preview form
```

Comment:
Not sure about this maybe this should be `preview-form-builder.rst` /cc @danrot what do you think it should be placed.